### PR TITLE
Add aXe test to OMBInfo component with modal open

### PR DESCRIPTION
**Pull request by ddzz**
## Description
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/224

## Acceptance criteria
- [ ] Unit tests pass on CI.

### CODE REVIEW

File: src/helpers/test-helpers.js [hunk 1]
```diff
@@ -32,7 +32,12 @@ export function axeCheck(component, ignoredRules = []) {
 
   div.innerHTML = '';
 
-  const mountedComponent = mount(component, { attachTo: div });
+  let mountedComponent;
+  if (state) {
+    mountedComponent = mount(component, { attachTo: div }).setState(state);
+  } else {
+    mountedComponent = mount(component, { attachTo: div });
+  }
```
Comment (bkjohnson) [inline src/helpers/test-helpers.js:40]: What about something like this:

```suggestion
  const mountedComponent = mount(component, attachTo: div});
  return state ? mountedComponent.setState(state) : mountedComponent;
```

File: src/components/OMBInfo/OMBInfo.unit.spec.jsx [hunk 2]
```diff
@@ -32,6 +32,16 @@ describe('<OMBInfo/>', () => {
         expDate="Expiration date"
       />,
     ));
+  it('should pass aXe check again', () =>
```
Comment (bkjohnson) [inline src/components/OMBInfo/OMBInfo.unit.spec.jsx:35]:
```suggestion
  it('should pass aXe check with modal open', () =>
```

File: src/components/OMBInfo/OMBInfo.jsx [hunk 3]
```diff
@@ -101,6 +101,7 @@ class OMBInfo extends React.Component {
           id="omb-modal"
           visible={this.state.modalOpen}
           onClose={this.closeModal}
+          title=""
```
Comment (bkjohnson) [inline src/components/OMBInfo/OMBInfo.jsx:104]: What is the impact of this?

Comment (ddzz) [reply src/components/OMBInfo/OMBInfo.jsx:104]: This was an artifact from manual testing, I'll take it out.

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**